doc/teacher.md

```diff
@@ -104,25 +104,23 @@ This approach prevents accidentally revealing commit history from old course ins
 - Promote them to your course's teacher on course members page
 
 Assistants will automatically be given organization `owner` role to be able to accept student enrollments, approve student groups and access all course repositories.
-They will also be added to the `allteachers` team.
 
 ## Student Enrollments
 
 Students enroll in your course by logging in on QuickFeed with their GitHub accounts, find the course among the course cards and click the `Enroll` link.
 You can access the full list of students (both already enrolled and those waiting for approval) in the `Members` menu of your course page, and accept their enrollments.
 
 After a student's enrollment has been accepted into a course, the student will have access to the course's `assignment` repository and the student's personal repository, e.g., `student-labs`.
-All students in a course will be added to the `allstudents` team in the course's GitHub organization.
 
 Note that the student may receive three invitation emails from `quickfeed-uis[bot]`.
```

Comment:
Is this still correct that students get three invitation emails?

Reply:
I don't think we need to add students to the `info` repository as it's a public repository.